www/api/og.tsx

```diff
@@ -0,0 +1,31 @@
+import { ImageResponse } from '@vercel/og';
+import React from 'react';
+
+export const config = {
+  runtime: 'experimental-edge',
+};
+
+export default function () {
+  return new ImageResponse(
+    (
+      <div
+        style={{
+          fontSize: 128,
+          background: 'white',
+          width: '100%',
+          height: '100%',
+          display: 'flex',
+          textAlign: 'center',
+          alignItems: 'center',
+          justifyContent: 'center',
+        }}
+      >
+        Hello world!
+      </div>
+    ),
+    {
+      width: 1200,
+      height: 600,
+    },
+  );
+}
```

Comment:
Couldn't get this working - unsure what to put in `vercel.json`

See https://vercel.com/docs/project-configuration

Also see https://vercel.com/docs/concepts/functions/edge-functions/og-image-generation

I have a feeling we'll to download URLs and base64-encode them, juding from [this example](https://vercel.com/docs/concepts/functions/edge-functions/og-image-examples#dynamic-text-generated-as-image)

Reply:
Oh shiiit it supports tailwind!!! https://vercel.com/blog/introducing-vercel-og-image-generation-fast-dynamic-social-card-images#tailwind-css-support

Reply:
If it's annoying to get working here I've also created https://github.com/trpc/og-image and invited you @iway1 

Might be easier to keep it there